index.src.html

```diff
@@ -199,9 +204,53 @@ <h3 id="examples">Examples</h3>
 <section>
   <h2 id="concept">Concepts</h2>
 
+  <h3 id="concept-client">Clients</h3>
+
+  A <dfn export>client</dfn> represents a particular origin's relationship to
+  a set of <a>endpoints</a>.
+
+  Each <a>client</a> has an <dfn for="client" export attribute>origin</dfn>,
+  which is an <a>origin</a>.
+
+  Each <a>client</a> has an
+  <dfn for="client" export attribute>endpoint-groups</dfn> list, which is a list
+  of <a>endpoint groups</a>, each of which MUST have a distinct
+  {{endpoint group/name}}.
```

Comment:
What happens when you have a name collision? We should define what the UA should do here: first wins, last wins, abort the header, etc.

Reply:
Done.

The parsing algorithm already mandated "keep first", but I added a sentence here reinforcing that.